_register.php

```diff
@@ -6,8 +6,11 @@
 use Shopware\OpenTelemetry\Instrumentation\ConnectionInstrumentation;
 use Shopware\OpenTelemetry\Instrumentation\DALInstrumentation;
 use Shopware\OpenTelemetry\Instrumentation\HttpClientInstrumentation;
+use Shopware\OpenTelemetry\Instrumentation\OpenSearchInstrumentation;
+use Shopware\OpenTelemetry\Instrumentation\ShopwareInstrumentation;
 use Shopware\OpenTelemetry\Instrumentation\SymfonyInstrumentation;
 use OpenTelemetry\SDK\Sdk;
+use Shopware\OpenTelemetry\Instrumentation\TwigInstrumentation;
 
 if (class_exists(Sdk::class) && Sdk::isInstrumentationDisabled(SymfonyInstrumentation::NAME) === true) {
     return;
```

Comment:
I'm not convinced that `SymfonyInstrumentation::NAME` is correct, and it should be 'shopware'.

but for now, I think it's fine to opt for leaving it as is.. not to break BC

Reply:
you can change it. :)